Assets/MixedRealityToolkit.Providers/WindowsVoiceInput/WindowsDictationInputProvider.cs

```diff
@@ -285,22 +283,6 @@ public override async void Disable()
             }
         }
 
-        /// <inheritdoc />
-        public override void Destroy()
-        {
-#if UNITY_EDITOR
-            if (UnityEditor.PlayerSettings.WSA.GetCapability(UnityEditor.PlayerSettings.WSACapability.Microphone))
-            {
-                UnityEditor.PlayerSettings.WSA.SetCapability(UnityEditor.PlayerSettings.WSACapability.Microphone, false);
```

Comment:
They never actually turned anything off. They only ran in editor and not on device. Any player setting changes were only made in play mode. When you exit play mode, you lose all your changes.

Also, we don't want to revert the capabilities. The UWP capabilities are primarily for building. They aren't something we want to turn on/off dynamically

Reply:
Services (used to? they still should, but maybe there's an issue) get instantiated at edit time. That's how things like the InputSystem are able to enforce specific `GazeProvider` scripts and input modules at edit time.

>Also, we don't want to revert the capabilities. The UWP capabilities are primarily for building. They aren't something we want to turn on/off dynamically

Why not? What if I add the spatial mapping service to check it out, then decide I don't want it? Now my app is shipping with a capability set that it doesn't use.

Reply:
Services do get instantiated/destroyed at edit time...until this PR though, DataProviders did not. So the destroy code originally was pointless. 

The problem going forward is Service.Destroy/Disable or even the profile inspectors aren't a sure way of determination for "hey I don't want this dataprovider/service/capability" anymore. 

You could be switching around between example scenes or have different scenes for different apps all in your Unity project. etc etc. 

Of course, this says the same about auto-enabling. If I open an example scene with eye tracking, it will auto-enable the capability even though my shipped scene doesn't use it.

The sure solution would be to check what profiles/scenes are being built and then update the capabilities according to that....but of course that is exceedingly difficult/possibly impossible.

The idea with this change though is more people end up in sucky situations where the capability is not set and they are completely unaware. All they see is the feature not working. Vs the alternative of getting a microphone permission popup once whether they may currently be using it or not.

An alternative I considered is to keep all the logic I have now....just don't automate any player settings configuration. Instead, I would just log warnings to the console that "hey users, you should enable this because it looks like you want to use it"

I'm fine with that alternative...it just seemed based on comments/feedback in multiple issues filed that people would prefer automation by default. And if someone doesn't want that, they can toggle the automation off in preferences

Reply:
>Services do get instantiated/destroyed at edit time...until this PR though, DataProviders did not. So the destroy code originally was pointless.

Ah! There's the issue.
Before data providers were moved to be handled by their services (which happened after this cleanup code was written), the MRTK object treated then basically the same and instantiated them at edit time.

@davidkline-ms

Reply:
Yes exactly. They were treated equally but then were treated separately with the Registry introduction. However, since their lifecycle was now managed by the services and not the MRTK gameobject and/or Registry, the current services only instantiated on enable, not init()

Reply:
Ah, my bad. Completely missed the part where this fixes that. Thanks!